contracts/cw20-dao-crowdfund/src/contract.rs

```diff
@@ -88,7 +89,7 @@ pub fn instantiate(
 
     let msg = SubMsg::reply_on_success(birth_msg, INSTANTIATE_FUNDING_TOKEN_REPLY_ID);
 
-    Ok(Response::default()
+    Ok(response
         .add_attribute("method", "instantiate")
```

Comment:
Lets add information about the fee manager contract that we're instantiated with here.